.github/CODEOWNERS

```diff
@@ -5,6 +5,6 @@
 /internal/resources/cloudprovider/*         @grafana/platform-monitoring @grafana/middleware-apps
 /internal/resources/connections/*           @grafana/platform-monitoring @grafana/middleware-apps
 /internal/resources/machinelearning/*       @grafana/platform-monitoring @grafana/machine-learning
-/internal/resources/oncall/*                @grafana/platform-monitoring @grafana/grafana-oncall
+/internal/resources/oncall/*                @grafana/platform-monitoring @grafana/grafana-irm-backend
```

Comment:
@grafana/platform-monitoring hello! would someone with Admin access to the repo please be able to add the `@grafana/grafana-irm-backend` team to have access to this repo? thanks!

Reply:
done